packages/remote-config/android/src/main/java/io/invertase/firebase/config/UniversalFirebaseConfigModule.java

```diff
@@ -53,12 +57,12 @@ Task<Void> fetch(String appName, long expirationDuration) {
     FirebaseApp firebaseApp = FirebaseApp.getInstance(appName);
 
     return Tasks.call(
-        getExecutor(),
-        () -> {
-          FirebaseRemoteConfig config = FirebaseRemoteConfig.getInstance(firebaseApp);
-          Tasks.await(expirationDuration == -1 ? config.fetch() : config.fetch(expirationDuration));
-          return null;
-        });
+      getExecutor(),
+      () -> {
+        FirebaseRemoteConfig config = FirebaseRemoteConfig.getInstance(firebaseApp);
+        Tasks.await(expirationDuration == -1 ? config.fetch() : config.fetch(expirationDuration));
+        return null;
+      });
```

Comment:
Are you using the same formatter as the project?

Reply:
I'm using IntelliJ, and it formatted the code like this (force of habit to hit the reformat code keybind). I'm not familiar with Java formatters. Could you point me to a command I can run to fix this formatting or should I manually revert the formatting changes?

Reply:
That's a good question actually, I can't see a script in the `package.json` for formatting android. What is the correct way @mikehardy?

Reply:
We (that is, myself on behalf of Invertase) repackage https://github.com/google/google-java-format in https://www.npmjs.com/package/google-java-format so we have autoformatting, and it is available here: https://github.com/invertase/react-native-firebase/blob/a58ea4bfd72c903d43844da30d4f3bf5feb0f057/package.json#L11

If you run that command locally until it finally succeeds (it may take quite a few runs, it is a bit unreliable unfortunately but it will always eventually run to completion) then all your files will be formatted locally to our standard

Reply:
Following @mikehardy's advice, I ran the formatter. It did change a few files, but these I think are now in sync with our code style